File: emihalic-projekt/emihalic_aplikacija_4/src/main/java/org/foi/nwtis/podaci/Korisnik.java
package org.foi.nwtis.podaci;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor()
public class Korisnik {

  @Getter
  @Setter
  private String korime;
  @Getter
  @Setter
  private String lozinka;
  @Getter
  @Setter
  private String ime;
  @Getter
  @Setter
  private String prezime;

  public Korisnik() {}
}

File: AIR2116-Development/core/src/main/java/hr/foi/air/core/Korisnik.kt
package hr.foi.air.core

import android.graphics.Bitmap
import java.util.*


data class Korisnik(
    var korisnickoIme: String,
    var lozinka: String,
    var ime: String,
    var prezime: String,
    var uloga: String,
    var pin: String,
    var id: String,
    var slikaURL: String,
    var slika: Bitmap? = null,
    var mobitel: String,
    var email: String,
    var datumRodenja: Date
)